### DISCUSSION TITLE
Mortal Kombat Specific Blood Feature

### DISCUSSION THREAD

**Discussion by ghost** [ikemen-engine/Ikemen-GO#850] [Feature Request]
Hi everyone! First of all, I don't feel like killing other fighters from around the MUGEN universe. But I wanted to see a Mortal Kombat blood feature. I personally like UMK3 blood sprites alot more. The blood feature can be found in the Options menu when you set the blood level from high, to medium, to low, or to none/off. However, if red sounds too gore, you can change the blood color to green.

**Comment by jankespro12** [comment]
Or white
